parambokeh/__init__.py

```diff
@@ -174,6 +174,9 @@ def __call__(self, parameterized, doc=None, plots=[], **params):
     def on_msg(self, msg):
         p_name = msg['p_name']
         p_obj = self.parameterized.params(p_name)
+        if isinstance(p_obj, param.Action):
+            getattr(self.parameterized, p_name)(self.parameterized)
+            return
```

Comment:
Other widgets won't be refreshed - is that a problem? What if the callback alters another parameter value? (I decided not to get into this now, deferring it to 'dependencies between parameters and code'...)

Reply:
>deferring it to 'dependencies between parameters and code'

+1

Reply:
Sounds fine.